exercises/concept/chessboard/chessboard_test.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require 'minitest/autorun'
+require_relative 'chessboard.rb'
+
+class ChessboardTest < Minitest::Test
+  def test_rank_range_is_a_range_of_integers
+    assert_equal 1..8, Chessboard.rank_range
+  end
+
+  def test_file_range_is_a_range_of_letters
+    assert_equal ?A..?H, Chessboard.file_range
```

Comment:
```suggestion
    assert_equal ('A'..'H'), Chessboard.file_range
```

Same thing, but probably more familiar to a student.  If we have introduced this earlier in some concept about range or strings, then this suggestion is not necessary to consider.

Reply:
I think this change is a good one!